project/FlywayConfiguration.scala

```diff
@@ -14,18 +16,12 @@
  * limitations under the License.
  */
 
-package za.co.absa.atum.server.future.model
-
-import java.util.UUID
+object FlywayConfiguration {
 
-case class ControlMeasure(
-  id: Option[UUID],
-  flowId: UUID,
-  partitionId: UUID,
-  metadata: ControlMeasureMetadata,
-  runUniqueId: Option[String] = None, // todo why not UUID?
-  checkpoints: List[Checkpoint] = List.empty
-) extends BaseApiModel {
+  val flywayUrl = "jdbc:postgresql://localhost:5432/atum_db"
```

Comment:
I'd probably split the host, port, and DB name into 3 separate vals and add them here via s-interpolation

Reply:
done